src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -44,28 +45,28 @@ export default class LollipopMutationPlot extends React.Component<ILollipopMutat
 
     readonly mutationAlignerLinks = remoteData<{[pfamAccession:string]:string}>({
         await: ()=>[
-            this.props.store.pfamGeneData
+            this.props.store.pfamDomainData
         ],
         invoke: ()=>(new Promise((resolve,reject)=>{
-            const regions = this.props.store.pfamGeneData.result.regions;
+            const regions = this.props.store.pfamDomainData.result;
             const responsePromises:Promise<Response>[] = [];
-            for (let i=0; i<regions.length; i++) {
+            for (let i=0; regions && i<regions.length; i++) {
                 // have to do a for loop because seamlessImmutable will make result of .map immutable,
                 // and that causes infinite loop here
                 responsePromises.push(
-                    request.get(`${getMutationAlignerUrl()}?pfamAccession=${regions[i].metadata.accession}`)
+                    request.get(`${getMutationAlignerUrl()}?pfamAccession=${regions[i].pfamDomainId}`)
```

Comment:
This might be a bit confusing. In ensembl biomart file `pfam domain accession` is named as `pfam domain id`, so in genome nexus we have also named it `pfamDomainId`